### ISSUE TITLE
Document and notify at start-up that wxPython 2.8.12.1 is the minimum supported version

### ISSUE THREAD

**Issue by yanne** [robotframework/RIDE#1180]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1141) by @pekkaklarck on 28 Nov 2012*

We are trying to decide which wxPython to require as the minimum version (<a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=1139" title="Document and test minimum wxPython and Python versions"> issue 1139 </a>). It seems that supporting 2.8.10 or older requires quite a bit of work and we would like to support only 2.8.11 and newer.

To be able decide can be ditch 2.8.10 and older, we need to find out how many users are still using them. We can (and will) send a mail asking about this, but the most efficient way is notifying people using these old versions about the possible removed support when RIDE starts. This needs to be done ASAP to be able to make an informed decision for RIDE 1.0.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1141#c1) by @pekkaklarck on 28 Nov 2012*


It would actually be best if we could make 2.8.12 (i.e. the latest release in the table 2.8 branch) the minimum version. Not sure can we actually do that, but the pop-up should be presented to all &lt; 2.8.12 users so that we can find that out.

The pop-up should probably only be shown when RIDE is run the first time.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1141#c2) by @mkorpela on 1 Dec 2012*


I think the pop-up should be shown every time when RIDE is opened.

When people see a notification only once and can ignore it from that point on, the notification will not be very effective.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1141#c3) by @mkorpela on 1 Dec 2012*


I think the pop-up should be shown every time when RIDE is opened.

When people see a notification only once and can ignore it from that point on, the notification will not be very effective.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1141#c4) by @pekkaklarck on 3 Dec 2012*


Agree with Mikko that pop-up should be always shown. That was implemented in
<a href="https://github.com/robotframework/RIDE/commit/a6a776dc5e31951e85f4e6f1728563b0d2d00731" rel="nofollow">https://github.com/robotframework/RIDE/commit/a6a776dc5e31951e85f4e6f1728563b0d2d00731</a>

Can you Mikko still test that the dialog is correctly shown when you actually have old wxPython version installed? I have only been testing that by changing the if statement to always display the dialog. Tatu ought to remove my `or True` that I left there earlier to make testing on different OSes easier.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1141#c5) by @pekkaklarck on 3 Dec 2012*


According to Mikko the dialog works fine with older wxPython versions. Closing.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1141#c8) by @pekkaklarck on 7 Dec 2012*


<a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=1139" title="Document and test minimum wxPython and Python versions"> Issue 1139 </a> has been merged into this issue.
